crates/types/src/config/common.rs

```diff
@@ -336,7 +345,7 @@ impl Default for CommonOptions {
             allow_bootstrap: true,
             base_dir: None,
             metadata_store_client: MetadataStoreClientOptions::default(),
-            bind_address: "0.0.0.0:5122".parse().unwrap(),
+            bind_address: None,
```

Comment:
It's quite nice that having explicit defaults results in config keys printing out when users run `restate-server --dump-config` - any thoughts on how we could preserve `bind-address` in the output key list? I think we'll still generate docs from the config schema so probably fine overall!

Reply:
when I run `restate-server --dump-config`, it will always have `bind-address` in it as it will always have a value if not set explicitly it will by derived and set by config loader
eg. 
config file only contains 
```toml
advertised-address = "http://127.0.0.1:5111/"
```
but `restate-server --dump-config` will print
``` bash
bind-address = "0.0.0.0:5111"
advertised-address = "http://127.0.0.1:5111/"
```
please elaborate if I misunderstood the question

Reply:
Oh cool, that's all I was asking for! I didn't test this and was wondering if your change perhaps caused `--dump-config` to not print the `bind_address` key.